o/opticall/opticall-0.8.1-foss-2022a.eb

```diff
@@ -0,0 +1,28 @@
+name = 'opticall'
+version = '0.8.1'
+
+homepage = 'https://github.com/molgenis/molgenis-pipelines'
+description = """This distribution contains several pipelines/protocols/parameter files,
+which you can use 'out-of-the-box' to align and impute your NGS data using MOLGENIS Compute."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+easyblock = 'MakeCp'
+
+#
+# Example url:https://bitbucket.org/tss101/opticall/get/master.tar.gz
+#
+
+source_urls = [('https://bitbucket.org/tss101/%s/get/' % (name))]
+sources = ['master.tar.gz']
```

Comment:
This fetches the master branch, which maybe something else tomorrow after new commits. Please check  if there are releases. If not you will need to fetch a specific commit based on the commit hash to make sure the result from this recipe is reproducible.